src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseHelpAndSupportSection.tsx

```diff
@@ -80,6 +81,14 @@ const ReleaseHelpAndSupportSection = ({
             <OfficialStatisticsSection />
           </AccordionSection>
         )}
+        {release.type.title === ReleaseType.AdHoc && (
```

Comment:
This may break some UI tests (not sure if they have been adjusted already) for publications using 'Ad hoc' as a release type. If you search for `id:help-and-support` in the robot tests directory you should be able to find the tests that may be affected by this change

Reply:
I ran them locally and I think it is ok. I did get a couple of failures but that seemed to be stuff that is unrelated. If it ends up breaking a test I'll commit a fix.